packages/shared/src/lib/flows/actions/action.ts

```diff
@@ -43,10 +42,10 @@ export type PieceActionSettings = {
   pieceName: string;
   actionName: string | undefined;
   input: Record<string, unknown>;
+  customizedInputs: Record<string, boolean>
```

Comment:
Can you make it something llike `renderInfo` and then place customize inside it's object, that way you can add more render information in the future.

Reply:
I think this should be added to piece triggers as well